CHANGELOG.md

```diff
@@ -46,9 +49,6 @@ The types of changes are:
 * Move tests into an `ops` subdir [#935](https://github.com/ethyca/fidesops/pull/935)
 * Move client code into an `ops` subdir [#964](https://github.com/ethyca/fidesops/pull/964)
 * Updated the docs docker base image to be consistent with the fidesops image [#949](https://github.com/ethyca/fidesops/pull/949)
-
-### Breaking Changes
-
 * Update fidesops to use bcrypt for hashing [#876](https://github.com/ethyca/fidesops/pull/876)
```

Comment:
From checking the records, we've ascertained this isn't necessarily a breaking change, and migration across versions is possible (while not explicitly supported).